src/libraries/Pool.sol

```diff
@@ -210,7 +210,7 @@ library Pool {
                 // current tick is below the passed range; liquidity can only become in range by crossing from left to
                 // right, when we'll need _more_ currency0 (it's becoming more valuable) so user must provide it
                 result = result
```

Comment:
Few notes here... From the code prior (not your changed code) I'm wondering why we need to add to result to begin with? result is not changed before this block of code.

Secondly, that made me think its odd to change the sign here and that technically it seems right to have `SqrtPriceMath.getAmount0Delta` and `SqrtPriceMath.getAmount1Delta` return the correctly signed delta especially since it is originally set there. Feels odd to set it incorrectly and then change it again here

Reply:
imo its actually a badly named function... the function returns a `uint256` thats described as
"amount0 Amount of currency0 required to cover a position of size liquidity between the two passed prices"
so its not a delta at all, its just an amount

So maybe it would make more sense to rename it `getAmount0`?

Otherwise I can do as youre saying - but we will need to change the return type of the function to actually make it a delta as it currently isnt one

Thoughts?

Reply:
> From the code prior (not your changed code) I'm wondering why we need to add to result to begin with? result is not changed before this block of code.

Agreed, updating

Reply:
> Secondly, ...
Also done